lib/location/location.c

```diff
@@ -325,20 +334,12 @@ int location_pgps_data_process(const char *buf, size_t buf_len)
 	return -ENOTSUP;
 }
 
-void location_cellular_ext_result_set(
-	enum location_ext_result result,
-	struct location_data *location)
-{
-#if defined(CONFIG_LOCATION_SERVICE_EXTERNAL) && defined(CONFIG_LOCATION_METHOD_CELLULAR)
-	location_core_cellular_ext_result_set(result, location);
-#endif
-}
-
-void location_wifi_ext_result_set(
+void location_cloud_location_ext_result_set(
 	enum location_ext_result result,
 	struct location_data *location)
 {
-#if defined(CONFIG_LOCATION_SERVICE_EXTERNAL) && defined(CONFIG_LOCATION_METHOD_WIFI)
-	location_core_wifi_ext_result_set(result, location);
+#if defined(CONFIG_LOCATION_SERVICE_EXTERNAL) && defined(CONFIG_LOCATION_METHOD_CELLULAR) &&\
+	defined(CONFIG_LOCATION_METHOD_WIFI)
```

Comment:
Shouldn't this be `defined(CONFIG_LOCATION_METHOD_CELLULAR) || defined(CONFIG_LOCATION_METHOD_WIFI)`?